test/live_mixer/live_queue_test.exs

```diff
@@ -0,0 +1,146 @@
+defmodule Membrane.LiveAudioMixer.LiveQueueTest do
+  use ExUnit.Case, async: true
+
+  alias Membrane.AudioMixer.LiveQueue
+  alias Membrane.AudioMixer.LiveQueue.Queue
+  alias Membrane.{Buffer, RawAudio, Time}
+
+  @stream_format %RawAudio{
+    channels: 1,
+    sample_rate: 48_000,
+    sample_format: :s16le
+  }
+  @duration Time.milliseconds(100)
+  @payload RawAudio.silence(@stream_format, @duration)
+  @buffer %Buffer{pts: 0, payload: @payload}
+  setup do
+    live_queue = LiveQueue.init(@stream_format)
+
+    %{live_queue: live_queue}
+  end
+
+  describe "Queues" do
+    test "adding", %{live_queue: live_queue} do
+      {:ok, %{queues: %{1 => queue}} = live_queue} = LiveQueue.add_queue(live_queue, 1)
```

Comment:
`LiveQueue.state_t` is an opaque so we shouldn't assert its internal form. Use functions provided by an API for testing. 

Having said that, I don't see much sense in splitting addition and deletion of buffers into separate tests